ts/src/pro/coinbasepro.ts

```diff
@@ -768,19 +770,8 @@ export default class coinbasepro extends coinbaseproRest {
             const ticker = this.parseTicker (message);
             const symbol = ticker['symbol'];
             this.tickers[symbol] = ticker;
-            const type = this.safeString (message, 'type');
-            const messageHash = type + ':' + marketId;
+            const messageHash = 'ticker:' + symbol;
```

Comment:
@ttodua as you can see using the watchMultiple approach, the code inside handleTicker is vastly simplified